scripts/build-env.sh

```diff
@@ -1,4 +1,8 @@
-#!/bin/sh
+#!/bin/bash
+
+REQUIRED_ENV_VARS=( GRAPHQL_URL GRIDPROXY_URL VERSION API_URL ACTIVATION_SERVICE_URL PLAYGROUND_URL RELAY )
```

Comment:
i think API_URL is confusing, let's name it substrate instead
```suggestion
REQUIRED_ENV_VARS=( GRAPHQL_URL GRIDPROXY_URL VERSION SUBSTRATE_URL ACTIVATION_SERVICE_URL PLAYGROUND_URL RELAY_URL )
```

Reply:
Done

Reply:
Done